app/src/main/java/com/github/fribourgsdp/radio/GameActivity.kt

```diff
@@ -152,6 +154,21 @@ open class GameActivity : AppCompatActivity(), GameView {
         startActivity(intent)
     }
 
+    private fun returnToMainMenu() {
+
+        if (isHost) {
+            playerGameHandler.database.disableGame(playerGameHandler.gameID)
```

Comment:
Here you should have simply done a function `playerGameHandler.disableGame`. That way you won't need to expose the database and the gameID.

It would simply be:
```kotlin
fun disableGame() {
    db.disableGame(gameID)
}
```

Reply:
Done! I also did similarly for the methods right below this which had a similar problem.